cfgov/unprocessed/js/organisms/Header.js

```diff
@@ -1,7 +1,23 @@
 // Required modules.
 import { checkDom, setInitFlag } from '../modules/util/atomic-helpers';
-import GlobalSearch from '../molecules/GlobalSearch.js';
-import MegaMenu from '../organisms/MegaMenu.js';
+import GlobalSearch from '../molecules/GlobalSearch';
+import MegaMenuOrig from '../organisms/MegaMenu';
+
+/*
+  TODO: Remove when 2019/2020 Mega Menu modifications are finalized.
+  Global variables are set by a flag in the base template.
+*/
+let MegaMenu;
+import MegaMenuVar1 from '../organisms/MegaMenuVar1';
+import MegaMenuVar2 from '../organisms/MegaMenuVar2';
+
+if ( window.cfpb && window.cfpb.megaMenuVar1 ) {
+  MegaMenu = MegaMenuVar1;
+} else if ( window.cfpb && window.cfpb.megaMenuVar2 ) {
+  MegaMenu = MegaMenuVar2;
+} else {
+  MegaMenu = MegaMenuOrig;
+}
```

Comment:
Do we need to import all three even if only one will ever be shown, and its selection is fixed in the page HTML?

Reply:
[Conditional imports](https://stackoverflow.com/questions/36367532/how-can-i-conditionally-import-an-es6-module#answer-46543835) are available, but it wraps the response in a Promise, which means we'd have to move all the code that follows in this file conditionally into the Promise block in some fashion. Given we'll strip these out (hopefully) soonish, I opted for the simpler importing them all. It inflates the JS a bit, but not terrible.